public/app/features/alerting/unified/mocks/server/handlers/alertmanagers.ts

```diff
@@ -135,9 +136,17 @@ export const updateAlertmanagerConfigHandler = (responseOverride?: typeof ALERTM
   });
 
 const getGrafanaAlertmanagerTemplatePreview = () =>
-  http.post('/api/alertmanager/grafana/config/api/v1/templates/test', () =>
-    // TODO: Scaffold out template preview response as needed by tests
-    HttpResponse.json({})
+  http.post<never, TemplatesTestPayload>(
```

Comment:
The real logic is more complex than this of course, but I just opted for making the endpoint return something relevant to the payload we sent in, so we can assert on it in the template preview